about.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="./css/global.css">
+  <link rel="stylesheet" href="./css/navsection.css">
+  <link rel="stylesheet" href="./css/aboutsection.css">
+  
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/bdd8845187.js" crossorigin="anonymous"></script>
+  
+  <title>About MoP.</title>
+</head>
+<body>
+<main>
+  <header>
+    <nav class="navigation d-flex align-items-center justify-content-between">
+      <i class="fa-solid fa-bars open-menu"></i>
+      <a href="./index.html">MoP.™</a>
+      <ul class="navigation__list">
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="index.html#program">Program</a></li>
+        <li><a href="index.html#speakers">Speakers</a></li>
+      </ul>
+    </nav>
+  </header>
+  <ul class="navigation__mobile">
+    <li><a href=""><img id="close-menu" class="close-menu" src="./images/close.png" alt="close"></a></li>
+     <li><a href="./index.html" class="navigation__mobile__links externalLink">Home</a></li>
+     <li><a href="index.html#program" class="navigation__mobile__links externalLink">Program</a></li>
+     <li><a href="index.html#speakers" class="navigation__mobile__links externalLink">Speakers</a></li>
+  </ul>
+  <section class="about">
+    <p class="about__para1 text-center">
+      "Greetings, students of Philosophy"
+    </p>
+    <h1>
+      Masters of Philosophy™ <br>
+      ANNUAL CONGRESS OF DISCOURSE <br>
+      2022 <br>
+    </h1>
+    <p class="about__para2">
+      The Congress brings together the most brilliant minds in modern Philosophy
+      to share knowledge and discuss about the latest advancements in the numerous 
+      fields of philosophy. We have philosphy professors from all over the globe 
+      who grace us with their attendance to give passionate lectures on the translations
+      of all our favorite ancient philosophers. 
+    </p>
+    <div class="p-3 text-center">
+      <h2 class="about__enq">
+        For any enquiries about the Congress of Discourse, 
+        feel free to contact us at:
+      </h2>
+      <a class="about__enq__mail" href="#">mastersofphilosophy@gmail.com</a>
+    </div>
+  </section>
+
+  <section class="statement d-flex flex-column">
+    <div class=" Ostate text-center ml-auto mr-auto">
+      Our Statement
+      <hr class="our-state"/>
+    </div>
+    <div class="statement__container">
+      <img class="rounded" src="./images/mission.jpg" alt="mission">
+      <img class="rounded" src="./images/vision.jpg" alt="vision">
+
+    </div>
```

Comment:
- [ ] Also to improve your UI design, you could add the background overlay color on your maps 👍 

- Your Design:

![photo42](https://user-images.githubusercontent.com/79058364/170608708-288cede8-091d-4198-af9f-00a6ca80f9a3.png)


- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![Annotation 2022-05-18 110524](https://user-images.githubusercontent.com/85113567/169014609-2f09ccf3-cab1-4dda-b381-8e449c302275.jpg)